tree/dataframe/src/RSqliteDS.cxx

```diff
@@ -60,11 +388,22 @@ constexpr char const *RSqliteDS::fgTypeNames[];
 RSqliteDS::RSqliteDS(std::string_view fileName, std::string_view query)
    : fDb(nullptr), fQuery(nullptr), fNSlots(0), fNRow(0)
 {
+   static bool isRegistered = Register();
    int retval;
 
-   retval = sqlite3_open_v2(std::string(fileName).c_str(), &fDb, SQLITE_OPEN_READONLY | SQLITE_OPEN_NOMUTEX, nullptr);
-   if (retval != SQLITE_OK)
-      SqliteError(retval);
+   // Opening the layer
+   if (IsURL(fileName)) {
+      if (!isRegistered)
+         throw std::runtime_error("Error");
```

Comment:
That should be, I guess, something like `"Processing remote files is not available. Please compile ROOT with Davix support to read from HTTP(S) locations."`